presentation/src/main/res/values/strings.xml

```diff
@@ -16,5 +16,6 @@
     <string name="skip_this_survey">Skip this survey</string>
```

Comment:
This is changed to "Skip this activity" to match MTB pause menu

Reply:
Fixed